types/index.d.ts

```diff
@@ -282,9 +286,9 @@ export class JoystickManager {
 }
 
 export interface Collection {
-    nipples: [];
-    idles: [];
-    actives: [];
+    nipples: any[];
+    idles: any[];
+    actives: any[];
```

Comment:
I'm not certain about it but wouldn't they be:

```suggestion
    actives: Joystick[];
```

Reply:
I've never had to access this class externally, so the type was never necessary from a user point of view.  This class was added just so the user can compile Typescript. The `any` type can be used (cast) to any matching type in Typescript, so it's like specifying the generic type `Object`, which is fine here.

Reply:
all three of them can be Joystick type... are you good with that?